rllib/env/wrappers/atari_wrappers.py

```diff
@@ -324,4 +325,5 @@ def wrap_deepmind(env, dim=84, framestack=True):
     # 4x image framestacking.
     if framestack is True:
         env = FrameStack(env, 4)
+    env.observation_space.atari_wrapped_space = original_space
```

Comment:
I don't get where `atari_wrapped_space` came from? Why is this not the `original_space` attribute that we already use in RLlib anyway?

Reply:
e.g. I mean `env.observation_space.original_space = original_space`?

Reply:
It's different from original space. We use original space for example to unflatten things afaik. But a wrapped environment potentially has one orignal space per wrapper. Or at least, it can have one inner-most space and one outer-most space of the wrapped environments. Since the original space of such an atari environment would not be the inner-most space (the space of the wrapped environment), we need to store this somehow to find out what the space of the wrapped environment was -> env.observation_space.atari_wrapped_space.

Again this is only because we auto-wrap atari environments.
We don't do this sort of stuff in any other situation so it's ok to tailor this I guess.